dist/function.py

```diff
@@ -0,0 +1,131 @@
+        'union ' : 2,
+        'enum ' : 3,
+        'WT_' : 4,
+        'wt_' : 5,
+        'double ' : 6,
+        'float ' : 7,
+        'size_t ' : 8,
+        'uint64_t ' : 9,
+        'int64_t ' : 10,
+        'uint32_t ' : 11,
+        'int32_t ' : 12,
+        'uint8_t ' : 13,
+        'int8_t ' : 14,
+        'u_int ' : 15,
+        'int ' : 16,
+        'u_char ' : 17,
+        'char ' : 18,
+        'bool ' : 19,
+        'va_list ' : 20,
+        'void ' : 21,
+
+    }
+    line = line.strip()
+    line = re.sub("^const ", "", line)
+    line = re.sub("^volatile ", "", line)
+
+    # Let WT_UNUSED terminate the parse. It often appears at the beginning
+    # of the function and looks like a WT_XXX variable declaration.
+    if re.search('^WT_UNUSED', line):
+        return False,0
+
+    # Let lines not terminated with a semicolon terminate the parse, if means
+    # there's some kind of interesting line split we probably can't handle.
+    if not re.search(';$', line):
+        return False,0
+
+    # Check for matching types.
+    for m in types:
+        # Don't list '{' as a legal character in a declaration, that's what
+        # prevents us from sorting inline union/struct declarations.
+        if re.search('^' + m + "\s*[\w(*]", line):
+            return True,types[m]
+    return False,0
+
+# Put function arguments in correct sort order.
+def function_declaration():
+    tmp_file = '__tmp'
+    for name in all_c_files():
+        skip_re = re.compile(r'DO NOT EDIT: automatically built')
+        s = open(name, 'r').read()
+        if skip_re.search(s):
+            continue
+
+        # Read through the file, and for each function, do a style pass over
+        # the local declarations. Quit tracking declarations as soon as we
+        # find anything we don't understand, leaving it untouched.
+        with open(name, 'r') as f:
+            tfile = open(tmp_file, 'w')
+            tracking = False
+            for line in f:
+                if not tracking:
+                    tfile.write(line)
+                    if re.search('^{$', line):
+                        r = [[] for i in range(30)]
+                        tracking = True;
+                    continue
+
+                found,value = function_args(line)
+                if value == -1:
+                    print >>sys.stderr, \
+                        name + ": illegal declaration: " + line.strip()
+                    sys.exit(1)
+                if value == 0 or not found:
+                    for arg in filter(None, r):
+                        for p in sorted(arg, key=function_args_alpha):
+                            tfile.write(p)
+                    tfile.write(line)
+                    tracking = False
+                    continue
+                r[value].append(line)
+
+            tfile.close()
+            compare_srcfile(tmp_file, name)
+
+# Report missing function comments.
+#missing_comment()
```

Comment:
Since we removed it from style.py, should we enable it here

Reply:
> Since we removed it from style.py, should we enable it here
 
Thanks!  Turned it off for debugging, forgot to turn in back on.

Fixed in 4b7f599.